libs/transactions/src/lib/components/transaction-item/transaction-item.component.ts

```diff
@@ -12,13 +18,12 @@ export class TransactionItemComponent implements OnChanges {
   amount = 0;
 
   ngOnChanges(changes: SimpleChanges): void {
-    if (changes.transaction) {
-      this.amount = this.transaction?.transaction.amountCurrency.amount ?? 0;
+    if (changes['transaction']) {
```

Comment:
You don't need to use square brackets when the name of the property of the object is just a word. Please use the dot notation.

Reply:
I changed because property 'transactions' comes from an index signature, so it must be accessed with ['transactions'].ts(4111)

I can disable noPropertyAccessFromIndexSignature if needed.